lms/djangoapps/django_comment_client/utils.py

```diff
@@ -181,10 +187,16 @@ def get_cached_discussion_id_map(course, discussion_ids, user):
     Returns a dict mapping discussion_ids to respective discussion xblock metadata if it is cached and visible to the
     user. If not, returns the result of get_discussion_id_map
     """
-    return get_cached_discussion_id_map_by_course_id(course.id, discussion_ids, user)
+    kwargs = {"course_id": course.id, "discussion_ids": discussion_ids, "user": user}
+    is_forum_community_ta = has_forum_access(
```

Comment:
Rather than passing `is_forum_community_ta` here, can we construct the variable where we need it? i.e. inside `get_cached_discussion_id_map_by_course_id`?